### PR TITLE
chore: switch to jv to determine & download jenkins versions

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

File: utils/release.bash [hunk 1]
```diff
@@ -121,7 +121,7 @@ function downloadAzureKeyvaultSecret(){
 # * <version> represents any valid existing version like 2.176.3 available at JENKINS_DOWNLOAD_URL
 # JENKINS_DOWNLOAD_URL: Specify the endpoint to use for downloading jenkins.war
 function downloadJenkinsWar(){
-  "$WORKSPACE"/utils/getJenkinsVersion.py
+  jv dowload
```
Comment (MarkEWaite) [inline utils/release.bash:124]: Spelling error?
```suggestion
  jv download
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Spelling error correction needed?

**Review by MarkEWaite — APPROVED** [review]

**Review by markyjackson-taulia — APPROVED** [review]

**Review by MarkEWaite — COMMENTED** [review]
I'm not seeing how the `jv` executable is installed into the workspace and added to the path for use in the script.  Is that done in some other location?

**Comment by timja** [comment]
> I'm not seeing how the `jv` executable is installed into the workspace and added to the path for use in the script. Is that done in some other location?

https://github.com/jenkins-infra/docker-packaging/pull/7

**Comment by MarkEWaite** [comment]
As far as I can tell, this is ready for merge.  If there are no objections, I'll merge it within the next 1-2 hours.

**Pull request merged**